docs/deployment/feature_gates.md

```diff
@@ -36,6 +36,7 @@ The following tables are a summary of the feature gates that you can set on diff
 | ShootManagedIssuer                 | `false` | `Alpha` | `1.93` |        |
 | VPAForETCD                         | `false` | `Alpha` | `1.94` |        |
 | VPAAndHPAForAPIServer              | `false` | `Alpha` | `1.95` |        |
+| AllowCredentialsBinding            | `false` | `Alpha` | `1.97` |        |
```

Comment:
```suggestion
| ShootCredentialsBinding            | `false` | `Alpha` | `1.98` |        |
```

Nit: Rename - We usually don't use verbs in feature gates (also see K8s feature gates).